### ISSUE TITLE
Cannot add score/place for two entries

### ISSUE THREAD

**Issue by jmlawler** [geoffhumphrey/brewcompetitiononlineentry#464]
I'm entering the scores for our competition and there are two entries, both in the same flight, where the entry score and place does not save. 

I access the entries through Scoring > Scores By Table > Enter Edit Scores for Table 'x'. Then entered all the scores for the flight and click 'Update.' The scores for the 3 other entries in the flight and all entries in the previous flight saved correctly, but I can't figure out why these two will not.

Do you have any suggestions?

**Comment by j031** [comment]
I'm not sure if this is the same issue but something similar happened to us last year, I don't know if the code has been fixed.  

We found a work around that if you sorted the data entry page so these were not in the same position it might work.  We found only some of the positions failed in some of the categories.  Not sure why.

Hope this helps.

**Comment by jmlawler** [comment]
Thanks for the reply. I gave this a try and was still unable to assign a score to the two entries. 

I'll keep digging around and see if I can find a solution.

**Comment by jmlawler** [comment]
Curiouser and curiouser.... 

Entering the scores from this weekends flights and an additional 3 entries would not record a score.

I once again tried j031's suggestion and changed the sort, but this time instead of simply selection another column to sort on I inverted the sort (desc) on the entry column. I entered the scores/mini-bos-place for the entries that were not accepting scores and clicked update.

The scores saved, but now those entries are duplicated in my 'score by category' and 'scores by table' views.

![2015-07-27_134839](https://cloud.githubusercontent.com/assets/13423010/8914719/68c73aa0-3466-11e5-9067-6f4f9568f9b4.png)
